specification/healthbot/resource-manager/Microsoft.HealthBot/stable/2022-08-08/healthbot.json

```diff
@@ -0,0 +1,947 @@
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      },
+      "delete": {
+        "operationId": "Bots_Delete",
+        "x-ms-examples": {
+          "BotDelete": {
+            "$ref": "./examples/ResourceDeletionDelete.json"
+          }
+        },
+        "description": "Delete a HealthBot.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "If resource is deleted successfully, the service should return 200."
+          },
+          "202": {
+            "description": "Accepted - Delete request accepted; operation will complete asynchronously"
+          },
+          "204": {
+            "description": "No content."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.HealthBot/healthBots/{botName}/listSecrets": {
+      "post": {
+        "operationId": "Bots_ListSecrets",
+        "x-ms-examples": {
+          "Bot List Secrets": {
+            "$ref": "./examples/ListSecrets.json"
+          }
+        },
+        "description": "List all secrets of a HealthBot.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "If secrets are retrieved successfully, the service should return 200 (OK).",
+            "schema": {
+              "$ref": "#/definitions/HealthBotKeysResponse"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.HealthBot/healthBots/{botName}/regenerateApiJwtSecret": {
```

Comment:
>regenerateApiJwtSecret [](http://example.com/codeflow?start=123&length=22)

Same for this

Reply:
@zizw123 I added the "x-ms-secret": true annotation. Looking into the Secrets section, that's the only thing that was missing. Please let me know if anything else is needed.